src/app/services/app-service/app.service.ts

```diff
@@ -39,6 +44,8 @@ export class AppStoryService implements IAppStoryService {
 
     const comments: Array<Observable<Comment>> = [];
 
+    if (!ids ) return forkJoin(comments);
```

Comment:
forkJoin => of

Reply:
that does not work, since of does not aggregate observables. but i think instead of the if else condition just use the elvis operator